site/_includes/layout.njk

```diff
@@ -7,6 +7,7 @@ siteTitle: Web platform features explorer
     <meta charset="utf-8">
     <title>{{ siteTitle }} - {{ title }}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" href="/assets/apexcharts.css">
```

Comment:
Could you move this line to `timeline.njk` instead? This way the CSS file is only loaded on the timeline page.